modules/sdk-coin-icp/src/icp.ts

```diff
@@ -79,8 +82,8 @@ export class Icp extends BaseCoin {
     throw new Error('Method not implemented.');
   }
 
-  isValidPub(_: string): boolean {
-    throw new Error('Method not implemented.');
+  isValidPub(HexEncodedPublicKey: string): boolean {
```

Comment:
1. Use `camelCase` naming
2. you don't know if the input argument is hex encoded here, the job of the func is to find out 
```suggestion
  isValidPub(key: string): boolean {
```

Reply:
changed